capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super.capitalize
```

Comment:
- [ ] Instead of `super.capitalize` kindly use `@nameable.correct_name.capitalize`